content/intro-to-storybook/react/it/conclusion.md

```diff
@@ -1,34 +1,34 @@
 ---
-title: 'Conclusion'
-description: 'Put all your knowledge together and learn more Storybook techniques'
+title: 'Conclusione'
+description: 'Metti insieme tutte le tue conoscenze e scopri altre tecniche di Storybook'
 ---
 
-Congratulations! You created your first UI in Storybook. Along the way you learned how to build, compose, test, and deploy UI components. If you’ve been following, your repo and deployed Storybook should look like this:
+Complimenti! Hai creato la tua prima UI in Storybook. Lungo il percorso hai imparato a costruire, comporre, testare e distribuire componenti UI. Se hai seguito, il tuo repository e Storybook distribuito dovrebbero assomigliare a questo:
```

Comment:
istead of **il tuo repository e Storybook distribuito** it should be **il tuo repository e distribuito Storybook**